go.mod

```diff
@@ -59,11 +63,11 @@ require (
 	google.golang.org/api v0.15.0
 	gopkg.in/ns1/ns1-go.v2 v2.0.0-20190322154155-0dafb5275fd1
 	gopkg.in/yaml.v2 v2.2.8
-	istio.io/api v0.0.0-20200324230725-4b064f75ad8f
-	istio.io/client-go v0.0.0-20200324231043-96a582576da1
-	k8s.io/api v0.17.5
-	k8s.io/apimachinery v0.17.5
-	k8s.io/client-go v0.17.5
+	istio.io/api v0.0.0-20200529165953-72dad51d4ffc
+	istio.io/client-go v0.0.0-20200529172309-31c16ea3f751
+	k8s.io/api v0.18.3
+	k8s.io/apimachinery v0.18.3
+	k8s.io/client-go v0.18.3
```

Comment:
Is it possible to keep these unrelated updates out of the PR?

Reply:
They are in a separate commit? That is about the most separation that is possible given they are required updates to use contour client v1.5.0 which is required for this PR to work.

Reply:
This has now been resolved as the dependency PR has been merged.